geode-core/src/main/java/org/apache/geode/internal/net/BufferPool.java

```diff
@@ -46,6 +48,18 @@ public BufferPool(DMStats stats) {
   private final ConcurrentLinkedQueue<BBSoftReference> bufferQueue =
```

Comment:
consider renaming "bufferQueue" since it now only contains buffers whose capacity is greater than DEFAULT_SOCKET_BUFFER_SIZE